docs/guides/qiskit-addons-cutting-gates.ipynb

```diff
@@ -0,0 +1,460 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8fd79758-2771-41ce-b17e-e8bfdf5b1023",
+   "metadata": {},
+   "source": [
+    "# Get started with circuit cutting using gate cuts\n",
+    "\n",
+    "This guide demonstrates two working examples of gate cuts with the `qiskit-addon-cutting` package. It covers using gate cutting to reduce the circuit width (the number of qubits) and circuit depth (the number of circuit instructions). The gate cutting enables reconstructing a four-qubit circuit using two two-qubit subexperiments.\n",
```

Comment:
```suggestion
    "This guide demonstrates two working examples of gate cuts with the `qiskit-addon-cutting` package. The first example shows how to reduce circuit depth (the number of circuit instructions) by cutting entangling gates on non-adjacent qubits that would otherwise incur a SWAP overhead when transpiled to hardware. The second example covers how to use gate cutting to reduce the circuit width (the number of qubits) by splitting a circuit into several circuits with fewer qubits. \n",
```